plugins/inputs/intel_powerstat/README.md

```diff
@@ -73,268 +110,279 @@ no per-CPU metrics are collected:
-| 0x2C         | Intel Westmere-EP               |
-| 0x2F         | Intel Westmere-EX               |
-| 0x2A         | Intel Sandybridge               |
-| 0x2D         | Intel Sandybridge-X             |
-| 0x3A         | Intel Ivybridge                 |
-| 0x3E         | Intel Ivybridge-X               |
-| 0x4E         | Intel Atom® Silvermont-MID      |
-| 0x5E         | Intel Skylake                   |
-| 0x55         | Intel Skylake-X                 |
-| 0x8E         | Intel KabyLake-L                |
-| 0x9E         | Intel KabyLake                  |
-| 0x6A         | Intel IceLake-X                 |
-| 0x6C         | Intel IceLake-D                 |
-| 0x7D         | Intel IceLake                   |
-| 0x7E         | Intel IceLake-L                 |
-| 0x9D         | Intel IceLake-NNPI              |
-| 0x3C         | Intel Haswell                   |
-| 0x3F         | Intel Haswell-X                 |
-| 0x45         | Intel Haswell-L                 |
-| 0x46         | Intel Haswell-G                 |
-| 0x3D         | Intel Broadwell                 |
-| 0x47         | Intel Broadwell-G               |
-| 0x4F         | Intel Broadwell-X               |
-| 0x56         | Intel Broadwell-D               |
-| 0x66         | Intel CannonLake-L              |
-| 0x57         | Intel Xeon® PHI Knights Landing |
-| 0x85         | Intel Xeon® PHI Knights Mill    |
-| 0xA5         | Intel CometLake                 |
-| 0xA6         | Intel CometLake-L               |
-| 0x8A         | Intel Lakefield                 |
-| 0x8F         | Intel Sapphire Rapids X         |
-| 0x8C         | Intel TigerLake-L               |
-| 0x8D         | Intel TigerLake                 |
-| 0xA7         | Intel RocketLake                |
-| 0x97         | Intel AlderLake                 |
-| 0x9A         | Intel AlderLake-L               |
-| 0xBE         | Intel AlderLake-N               |
-| 0xB7         | Intel RaptorLake                |
-| 0xBA         | Intel RaptorLake-P              |
-| 0xBF         | Intel RaptorLake-S              |
-| 0xAC         | Intel MeteorLake                |
-| 0xAA         | Intel MeteorLake-L              |
-
-### uncore frequency
-
-Note that only certain processors support the uncore frequency module as well:
-
-| Model number | Processor name                  |
-|--------------|---------------------------------|
-| 0x55         | Intel Skylake-X                 |
-| 0x6A         | Intel IceLake-X                 |
-| 0x6C         | Intel IceLake-D                 |
-| 0x47         | Intel Broadwell-G               |
-| 0x4F         | Intel Broadwell-X               |
-| 0x56         | Intel Broadwell-D               |
-| 0x8F         | Intel Sapphire Rapids X         |
-| 0xCF         | Intel Emerald Rapids X          |
+    - `cpu_c0_state_residency`
+    - `cpu_c1_state_residency`
+    - `cpu_c3_state_residency`
+    - `cpu_c6_state_residency`
+    - `cpu_c7_state_residency`
+    - `cpu_busy_cycles`
```

Comment:
```suggestion
```
Or indicate it is deprecated.

Reply:
Marked as deprecated as part of commit [e2de6f6b](https://github.com/influxdata/telegraf/pull/14363/commits/e2de6f6b2577fe4e9ec7b420132b2750fe7e2654).